docs/simulation/calculations.yaml

```diff
@@ -147,21 +147,36 @@ POPs:
       RGO wages:
         Owners:
           if minimum worker wages > normal worker wage:
-            formula: (RGO income - total worker income) * owner POP size / owners * (1 - effective tax) 
+            formula: (RGO income - total worker income) * owner POP size / n_owners * (1 - effective tax) 
           else:
-            formula: RGO income * min(0.5, 2 * owners / workers) * owner POP size / owners * (1 - effective tax) 
-            owners: total size of owner POPs in state
-            workers: number of employed workers in RGO
+            formula: RGO income * min(0.5, 2 * n_owners / n_workers) * owner POP size / n_owners * (1 - effective tax) 
+            n_owners: total size of owner POPs in state
+            n_workers: number of employed workers in RGO
             min(0.5: Hardcoded maximum of half the RGO income.
-            2*: Hardcoded value.
+            2 *: Hardcoded value.
         Workers:
           if is_slave: 0
           else:
             if minimum wage > normal wage:
               formula: min(minimum wage, RGO income * employed workers in POP / total non-slave employed workers) * (1 - effective tax) 
             else:
-              formula: RGO income * max(0.5, 1 - 2 * owners / workers) * employed workers in POP / total non-slave employed workers * (1 - effective tax) 
-              owners: total size of owner POPs in state
-              workers: number of employed workers in RGO
+              formula: RGO income * max(0.5, 1 - 2 * n_owners / n_workers) * employed workers in POP / total non-slave employed workers * (1 - effective tax) 
+              n_owners: total size of owner POPs in state
+              n_workers: number of employed workers in RGO
               max(0.5: Hardcoded minimum of half the RGO income.
-              2*: Hardcoded value.
\ No newline at end of file
+              2 *: Hardcoded value.
+Scores:
+  Industrial power:
+    formula: sum of investment scores + sum of state factory scores
+    investment score: money_invested * defines.country.INVESTMENT_SCORE_FACTOR / 100
+    state factory score:
+      if n_workers > 0:
+        formula: sum_factory_levels_in_state * max(0.2, min(4, floor(n_workers_in_state / 100) * 400 / sum_workforce_in_state))
```

Comment:
When you have lower and upper bounds for a value you could use `clamp(value, low, high)` instead of `max(low, min(high, value))`. It's a common function in languages like [C++](https://en.cppreference.com/w/cpp/algorithm/clamp) and especially in shader languages, and IMO it's easier to read.

Reply:
I am aware of `clamp`. I avoid using it as there are different orders for the arguments per language.
css for example uses `clamp(min, val, max)`. `min` and `max` are unambiguous.